source/onboard/ad-ldap-groups-synchronization.rst

```diff
@@ -348,6 +348,6 @@ Only users that are members of groups synchronized to team are able to discover
 Why don't users get readded to teams or channels once they have been removed from and then later re-added to the LDAP group?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The implementation of group removals does not currently differentiate between users who have removed themselves or have been removed by the LDAP synchronization process. Our design optimizes for users who have removed themselves from a team or channel. In the future, we may add the ability for Admins to re-add users who have been removed, and even prevent users from leaving, a team or channel.
+The implementation of group removals does not differentiate between users who have removed themselves or have been removed by the LDAP synchronization process. Our design optimizes for users who have removed themselves from a team or channel. In the future, we may add the ability for Admins to re-add users who have been removed and even prevent users from leaving a team or channel.
 
-Additionally, LDAP users who are not accessible to Mattermost based on filters will be removed from the groups and from group synced teams and channels. If they were removed from teams and channels then they will not be re-added to those teams and channels upon becoming subsequently reaccessible to Mattermost.
+Additionally, LDAP users who are not accessible to Mattermost based on filters will be removed from the groups and synced teams and channels associated with them. If they were removed from teams and channels, they would not be re-added to those teams and channels upon becoming subsequently reaccessible to Mattermost.
```

Comment:
```suggestion
Additionally, LDAP users who are not accessible to Mattermost based on filters will be removed from the groups and from group synced teams and channels. If they were removed from teams and channels then they would not be re-added to those teams and channels upon becoming subsequently reaccessible to Mattermost.
```

Reply:
I've reverted parts of this as the original wording describes very specific behavior related to groups as well as group synced teams and channels (two different things).

Reply:
Okay.